syntaxes/log.tmLanguage

```diff
@@ -39,7 +39,7 @@
 			</dict>
 			<dict>
 				<key>match</key>
-				<string>\b(ERROR|FAILURE|FAIL)\b</string>
+				<string>\b(ERROR|EROR|FAILURE|FAIL)\b</string>
```

Comment:
Sorry for the delay, looks like I forgot to send the review. I wrote a comment some time ago but suspect you didn't see it.